warehouse/transformer/types.go

```diff
@@ -13,7 +13,8 @@ import (
 
 type (
 	Transformer struct {
-		now func() time.Time
+		now           func() time.Time
+		uuidGenerator func() string
```

Comment:
uuidGenerator -> generateUuid ?

Reply:
`t.generateUuid()` sounds more preferable for a standalone function, not a functional field. Since it's being injected into a struct for customization or testing, I would prefer it to be a noun rather than a verb.